integration-tests/environments/react-native/android/app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,8 @@
       android:icon="@mipmap/ic_launcher"
       android:roundIcon="@mipmap/ic_launcher_round"
       android:allowBackup="false"
-      android:theme="@style/AppTheme">
+      android:theme="@style/AppTheme"
+      android:usesCleartextTraffic="true">
```

Comment:
[Non-blocker]  Is this change intentional?  Do we need to use non-HTTS traffic?  ([reference](https://developer.android.com/guide/topics/manifest/application-element) (massive webpage without anchors :( ))

Reply:
I’ll double check that it’s needed though